### ISSUE TITLE
New users using the "authored by" radio button should get a warning

### ISSUE THREAD

**Issue by ngoldbaum** [lobsters/lobsters#1186]
If a new user's blog has been seen before there's nothing adding friction to doing self-promotion as a new user. See e.g. this domain:

https://lobste.rs/domains/allaboutcoding.ghinda.com

EDIT: (since submitting the issue the second story got deleted, https://lobste.rs/s/qyxaya/where_promote_article_about_ruby)

If a new user clicks the "authored by" button, they should probably get the same warning a new user submitting an unseen domain gets. Of course they could still resubmit without the "authored by" but at least well-intentioned users will get a warning.

**Comment by pushcx** [comment]
To make sure I follow - you're thinking of this validation?

> [example.com] is an unseen domain from a new user. We restrict this to discourage self-promotion and give you time to learn about topicality. Skirting this with a URL shortener or tweet or something will probably earn a ban.

I'm asking because it's a Story validation, so the new user can't submit their link at all. If this warning you're thinking of is added it'd be a bit of a policy change, right now users are allowed to submit links to their own stuff if someone has submitted the domain before (...or it's github, youtube, or another popular site for self-publishing). We do [explicitly encourage authors](https://github.com/lobsters/lobsters/blob/a2831a81ca1e23fb374789b6dd950af76a8429e6/app/views/about/about.html.erb#L117) who've had their work submitted to ask to join the site. Right now people just get treated pretty skeptically if the line under the headline says "by" instead of "via" and the username is green.

I can see how that would be an appropriate rule change. It feels like we've seen a significant increase in people using the site exclusively for self-promo this year.

A softer approach would be to add text to the submission page (similar to the one that detects and suggests users remove site names), it could explain that they may want to wait until they've first built up some trust with the community by posting comments and submitting other people's interesting stuff. That would continue to permit authors who get invited to immediately post. Though I'm not sure if that's worth keeping; typically they find their way to the site while an article of theirs is already on the home page; spacing out more from their site might also be an improvement.

**Comment by ngoldbaum** [comment]
Yeah, I guess an “are you *sure*?” check that links to a page describing community norms before letting them submit would also work.

All of this is predicated on someone clicking the “authored by” button, which is optional, so a policy change probably isn’t warranted some additional friction would at least tell well-intentioned but ignorant submitters that they are doing something that might raise eyebrows if they’re not also contributing to the site in other ways.
